Sources/FluentBenchmark/Tests/GroupByTests.swift

```diff
@@ -0,0 +1,28 @@
+extension FluentBenchmarker {
+    public func testGroupBy() throws {
+        try self.testGroupBy_join()
+    }
+
+    private func testGroupBy_join() throws {
+        try runTest(#function, [
+            SolarSystem()
+        ]) {
+            let earth = try XCTUnwrap(Planet.query(on: self.database).filter(\.$name == "Earth").first().wait())
+
+            let planets = try Planet.query(on: self.database)
```

Comment:
as mentioned in Discord, this test _is_ a bit of a stretch, but if I was going to _try_ to attribute meaning to this query so that I could read the test case as silly as opposed to confusing, I think I would name this "planetTags" because we don't end up with "two earth planets" but rather "two planet/tag relationships where the planet is earth"

Reply:
Good idea!